components/form-builder/app/Settings.tsx

```diff
@@ -69,6 +70,22 @@ const HintText = ({ id, children }: { id: string; children?: JSX.Element | strin
   );
 };
 
+const InvalidEmailError = ({ isActive }: { isActive: boolean }) => {
+  const { t } = useTranslation("form-builder");
+
+  return isActive ? (
+    <div className="mt-2 mb-2" role="alert">
```

Comment:
The alert area should be present on the page before it is filled with content. See this bit in the [aria alert docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role): 

> As with all other [live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions), alerts will only be announced when the content of the element with role="alert" is updated. Make sure that the element with the role is present in the page's markup first

So the outer container with `role="alert"` should be in the dom but invisible, and then populated with the message when triggered.

Reply:
Also could add a `aria-invalid` attribute to the input when the error is triggered https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid